apps/patient-documents/src/pages/styles/Index.module.scss

```
@@ -0,0 +1,20 @@
+@use "@carbon/react/scss/spacing" as *;
+@use "@carbon/react/scss/theme" as *;
+
+.page {
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-05;
+  // Clear the fixed Carbon header ($spacing-09), then the gap below it ($spacing-06).
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)